components/Error.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Layout from './Layout'
+import Navlink from './NavLink'
+import { Text } from './theme/Text'
+type ErrorProps = {
+  title: string
+  src: string
+  message?: string
+}
+const Error: React.FC<ErrorProps> = ({ title, message, src }) => {
+  return (
+    <Layout>
+      <div className="container">
+        <div className="row">
+          <div className="d-flex col-sm-3 align-items-center justify-content-center">
+            <div className="text-center">
+              <Text size="xl" bold={true}>
+                {title}
+              </Text>
+              <Text size="md">{message}</Text>
+              <Navlink path="/" className="btn btn-primary py-3 px-5 mt-3">
```

Comment:
We have a button component made already
https://github.com/garageScript/c0d3-app/blob/master/components/theme/Button.tsx

Reply:
I remade Error component with Button (see storybook above). It looks a bit small on big screen but neither Button not Text components allow bigger padding/margins.

Reply:
Our button component does seem too limited for the context of the error component. You could use a `div` to wrap the button in order to add margin, and also wrap the text inside with an html element(`like h1`) to change the font size. But then the size of the button would still be too small even with the largest size available. Considering we would have to make an `xl` class for the button component too, I think the code would be more simple and the UI would look better with the Navlink component!

Reply:
OK, I reverted it back to NavLink, see storybook.